### ISSUE TITLE
Cannot install 0.3.x from crates.io

### ISSUE THREAD

**Issue by NathanHowell** [mechiru/googapis#14]
There are multiple versions of 0.3.0 and 0.3.1... `cargo` downloads one version and validates against the other one. this is a long standing bug, but basically you just need to bump the version number every publish.

```console
> cargo build
  Downloaded googapis v0.3.1+20200930
error: failed to verify the checksum of `googapis v0.3.1+20200930`
```

see for more info https://github.com/rust-lang/crates.io/issues/1059

**Comment by mechiru** [comment]
@NathanHowell 
Thank you for this report.
I've published v0.3.3. Try it.

**Comment by NathanHowell** [comment]
it works now, thanks!
